waiter/test/waiter/state_test.clj

```diff
@@ -868,6 +868,10 @@
                           :failed-instances [{:message "Memory limit exceeded:" :flags #{:memory-limit-exceeded}}
                                              {:message "Memory limit exceeded:" :flags #{:memory-limit-exceeded}}]
                           :expected :not-enough-memory}
+                         {:name "health-check-misconfigured" :healthy-instances [] :unhealthy-instances []
```

Comment:
I would favor either one entry per line or commas between entries if on the same line